rslib/src/browser_table.rs

```diff
@@ -53,13 +77,13 @@ pub struct Font {
 }
 
 trait RowContext {
-    fn get_cell_text(&mut self, column: &str) -> Result<String>;
+    fn get_cell_text(&mut self, column: &Column) -> Result<String>;
```

Comment:
Minor thing: as Column is a Copy type and smaller than a pointer, it's more ergonomic and faster to just pass it directly instead of passing a reference